microsoft-edge/progressive-web-apps-chromium/how-to/microsoft-store.md

```diff
@@ -87,35 +87,72 @@ Your download is a `.zip` archive that contains an `.msixbundle` file and a `.cl
+
+The following figure shows the UI that is introduced when a user moves outside the scope of a PWA: 
+
+:::image type="content" source="../media/locale-redirection-additional-ui.png" alt-text="The additional UI with URL and page title when the app is redirected to another domain.":::
+
+### Domain redirection with browser-installed PWAs
+
+A Web App Manifest is tied to a single domain.  However, some PWAs use locale-specific domains for their customers in specific regions of the globe.  When visiting the PWA in a web browser, customers are seamlessly transitioned from the principal domain (for example, contoso.com) to a locale-specific domain (for example, contoso.co.ke), because the redirect happens during initial load of that website.
+
+Customers who install the PWA from Microsoft Edge would therefore install the PWA from the locale-specific domain.  Subsequent launches of the PWA go directly to that locale-specific domain, instead of first going to the principal domain.
+
+### Domain redirection with PWAs installed from the Microsoft Store
+
+PWAs that are installed from the Microsoft Store have a hard-coded start URL that is pointed at the principal domain.  When the PWA is launched, the PWA initially navigates to the principal domain, and then a customer may (as necessary) be redirected to their locale-specific domain. If that redirection occurs, the navigation is considered "out of scope".  As a result, the app displays the URL and page title at the top of the page.
+
+Displaying the URL and page title is a security feature to ensure that users know they have left the context of the PWA.  This added UI makes sense when a user loads a page from another website in the context of the PWA.  However, that added UI may be inappropriate when a user moves among domains that are all part of the same website.
+
+### Prevent the locale-specific URL and title from being displayed
+
+To prevent the additional UI from being shown in a PWA that's installed from the Microsoft Store, you can use [URL Handlers](https://github.com/WICG/pwa-url-handler/blob/main/explainer.md) to enable the PWA to span multiple locale-specific domains.
+
+To prevent displaying the URL and page title:
+
+1. Within the PWA's Web App Manifest, use [the `url_handlers` member](https://github.com/WICG/pwa-url-handler/blob/main/explainer.md#manifest-changes) to specify an array of origins that are associated with that app.
+1. On each of the referenced origins, include a [`web-app-origin-association` file](https://github.com/WICG/pwa-url-handler/blob/main/explainer.md#web-app-origin-association-file) that associates the PWA with that domain.
```

Comment:
@aarongustafson - in the linked example, is Example 1's intro sentence explaining the JSON that follows it? It's unclear to me in Example 1 what "web-app-origin-association file" refers to.

Reply:
Yes. Example 1 shows what the web-app-origin-association file (which is JSON) should look like in each domain.